src/renderer/preload.d.ts

```diff
@@ -25,6 +25,7 @@ declare global {
         fetch(): Promise<ConnectionModel[]>;
         create(model: ConnectionModel): Promise<ConnectionModel>;
         select(id: number): null;
+        delete(id: number): Promise<null>;
```

Comment:
```suggestion
        delete(id: number): Promise<void>;
```

Since you're not returning anything, it should be void. null actually returns null, I changed this for select in my pr as well